tests/bson-serialize-object-serializable-006.phpt

```diff
@@ -1,5 +1,7 @@
 --TEST--
 BSON serialization: PHP serializable object: examples from spec
+--SKIPIF--
+<?php if (!defined("HHVM_VERSION_ID")) exit("skip PHPC encodes __pclass first"); ?>
```

Comment:
Fwiw, the protected/private tests in this file should pass with https://github.com/mongodb-labs/mongo-php-driver-prototype/pull/118; however, `__pclass` is also here so we need to skip the entire file.